helmenv.sh

```diff
@@ -154,6 +154,10 @@ function helmenv_uninstall(){
 function helmenv_list(){
     installed_versions="$(find "${HELM_BINARY_PATH}"/ -name '*helm-*' -printf '%f\n' | grep -Eo 'v([0-9]\.?)+$' | sed '/^$/d' | sort --version-sort)"
     echo "$installed_versions"
+    if [[ -n $installed_versions ]]; then
+        echo
+        helm version -c
+    fi
```

Comment:
This breaks the `fzf` functionality in [`helmenv_use`](https://github.com/alexppg/helmenv/pull/11/files#diff-40a8084ede4c21c311ad91386dbbe9b7R167), since it uses the output of `helmenv_list` to populate the list of helm versions to choose from for `fzf`.

Reply:
ok, it does not make sense then to add this feature, as I can use `helm version -c`

Reply:
I agree, it's not very useful to have a separated command to see the version. But I was thinking about adding an asterisk in the used version in `helmenv_list`.